default.py

```diff
@@ -47,6 +49,14 @@ def addon_log(string):
     msg = '%s: %s' % (LOGGING_PREFIX, string)
     xbmc.log(msg=msg, level=xbmc.LOGDEBUG)
 
+def ShowBusyDialog():
+    busydialog.create()
+
+def HideBusyDialog():
+    try:
+        busydialog.close()
+    except RuntimeError,e:
```

Comment:
Did you encounter errors with showing/hiding the busy dialog?

If not, then I think you can do away with the entire wrapping function idea and just use busydialog.create/close() directly everywhere.